tests/test_sourmash.py

```diff
@@ -556,7 +556,7 @@ def test_compare_containment_require_scaled(c):
 def test_do_plot_comparison(c):
     testdata1 = utils.get_test_data('short.fa')
     testdata2 = utils.get_test_data('short2.fa')
-    c.run_sourmash('compute', '-k', '31', testdata1, testdata2)
+    c.run_sourmash('sketch', 'translate', '-p', 'k=31,num=500', testdata1, testdata2)
```

Comment:
note: technically, 'compute' without any parameters is the same as `sourmash sketch dna`, but it doesn't seem to matter for these tests so that's fine!

Reply:
Should I change it then?